### DISCUSSION TITLE
Problem with keys and mod

### DISCUSSION THREAD

**Discussion by UKUNIFOREVER** [maruohon/litematica#653] [General]
The mod used to work for me, but now it doesn't.
Previously, the M key opened a menu in which I could easily select my schemes and place them
Currently, I can’t do this, since initially the scheme selection is on the S key, and I don’t know and don’t want to configure the mod myself
I tried installing older versions of Malilib and Litematica, but that didn't help.
I can send a screenshot or video proof

Screenshot
https://drive.google.com/file/d/1tjrjhF6Ddt5UOe42okiuvK9dpuBeaS9y/view?usp=share_link

**Comment by maruohon** [comment]
Did you change some of the hotkeys? Or do you now use a non-QWERTY keyboard layout?

Does `M + C` still work to open the config menu? You can also access the config menu via Mod Menu's mod list, if needed, and assign a new hotkey to open the main menu.

**Reply by UKUNIFOREVER** [reply]
the fact is that I changed the keys after this problem
Roughly speaking, my mod out of the box doesn’t work that way, and I don’t understand what the problem is
I don’t even know how to correct it, I can’t even understand what the problem is, because before the mod worked well, it was probably in September, somewhere like that
everyone I saw this mod did not have such a problem

**Reply by maruohon** [reply]
I still don't quite understand, does the mod still at least load and work otherwise? It's just the hotkeys that are somehow messed up? And you are still able to re-bind the hotkeys?
